NEWS.md

```diff
@@ -1,6 +1,9 @@
 
 # renv 1.1.0  (UNRELEASED)
 
+* If `library` is unsupplied, renv functions default to using the project
+  library, if `project` is supplied and the project library exists.
```

Comment:
This reads pretty awkwardly to me. Can we give a more concrete example of the consequences of this change, or where users might see a change in behavior?